source/index.html.md.erb

```diff
@@ -30,13 +30,13 @@ When you use some of our APIs, you have to submit HTTP fraud prevention headers.
 We work with you to help meet this specification. If after discussions with 
 HMRC an application continues to submit incorrect or missing data, software 
 providers may be fined and blocked from using HMRC APIs.
-Check the <a href="Fraud%20Prevention%20Header%20Data%20Compliance%20and%20Sanctions%20Guidelines.pdf" onclick="ga('send', 'event', 'On Page Navigation', 'Why you must send data', 'Compliance and Sanctions Guidelines');">Compliance and Sanctions Guidelines</a>.
+Check the <a href="Fraud%20Prevention%20Header%20Data%20Compliance%20and%20Sanctions%20Guidelines.pdf" onclick="ga('send', 'event', 'On Page Navigation', 'Why you must send data', 'Compliance and Sanctions Guidelines');">Compliance and Sanctions Guidelines (PDF)</a>.
 
 
 ### Privacy and security
 
 HMRC has the right to collect audit data. We follow best practices set out by the Information Commissioner’s Office.
 
-Transaction monitoring is a key security approach used in the UK and globally. Our approach follows the National Cyber Security Centre (NCSC) and the Cabinet Office’s <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/271268/GPG_53_Transaction_Monitoring_issue_1-1_April_2013.pdf" onclick="ga('send', 'event', 'On Page Navigation', 'Privacy and security', 'Recommended guidance');">recommended guidance</a>.
+Transaction monitoring is a key security approach used in the UK and globally. Our approach follows the National Cyber Security Centre (NCSC) and the Cabinet Office’s <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/271268/GPG_53_Transaction_Monitoring_issue_1-1_April_2013.pdf" onclick="ga('send', 'event', 'On Page Navigation', 'Privacy and security', 'Recommended guidance');">recommended guidance (PDF)</a>.
```

Comment:
Looking at the [JIRA ticket](https://jira.tools.tax.service.gov.uk/browse/CMQ-2200), it seems that the link text should be:
> National Cyber Security Centre (NCSC) and the Cabinet Office’s recommended guidance (PDF)

Reply:
sorted